graylog2-web-interface/src/components/configurations/TimeRangePresetForm.tsx

```diff
@@ -136,12 +135,6 @@ const TimeRangePresetForm = ({
     [onUpdate, options],
   );
 
-  const { config } = useSearchConfiguration();
-  const limitDuration = useMemo(
-    () => moment.duration(config?.query_time_range_limit).asSeconds() ?? 0,
-    [config?.query_time_range_limit],
-  );
```

Comment:
As far as I can tell, it did not make sense to calculate the `limitDuration` here in the previous version, because we were going to ignore it anyway because of `ignoreLimitDurationInTimeRangeDropdown` from the `TimeRangeInputSettingsContext`.

With the changes in this PR we can just set `limitDuration` to `0` in this component.